pyscriptjs/src/components/pyconfig.ts

```diff
@@ -1,11 +1,25 @@
-import * as jsyaml from 'js-yaml';
 import { BaseEvalElement } from './base';
-import { appConfig } from '../stores';
+import { appConfig, addInitializer, runtimeLoaded } from '../stores';
 import type { AppConfig, Runtime } from '../runtime';
-import { PyodideRuntime, DEFAULT_RUNTIME_CONFIG } from '../pyodide';
+import { PyodideRuntime } from '../pyodide';
 import { getLogger } from '../logger';
+import { readTextFromPath, handleFetchError, inJest, mergeConfig, validateConfig } from '../utils'
+
+// Premise used to connect to the first available runtime (can be pyodide or others)
```

Comment:
SP: promise?

Reply:
aah, we can change it, unless it really meant `premise`, not sure tbh, because we don't use `await` stuff here

Reply:
it's more of a consumer from a global store here I guess

Reply:
changing it to `subscriber`